.github/workflows/build.yml

```diff
@@ -136,16 +143,50 @@ jobs:
 
   features:
     name: Features
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    # XXX XXX TBD SEE IF THIS WORKS IN GITHUB CI:
+    if: ${{ matrix.rust == 'nightly' || matrix.os == 'ubuntu-latest' }}
```

Comment:
aargh does not seem to work: https://github.com/brody4hire/portable-rustls/actions/runs/13269148730

> [Invalid workflow file: .github/workflows/build.yml#L148](https://github.com/brody4hire/portable-rustls/actions/runs/13269148730/workflow)
> The workflow is not valid. .github/workflows/build.yml (Line: 148, Col: 9): Unrecognized named-value: 'matrix'. Located at position 1 within expression: matrix.rust == 'nightly' || matrix.os == 'ubuntu-latest'